src/infra/nodemailer/setup.js

```diff
@@ -0,0 +1,41 @@
+import nodemailer from 'nodemailer';
+import * as dotenv from 'dotenv';
+import { generatingQRCode } from '../../utils/generateQRCode.js';
```

Comment:
Adicionar uma linha em branco entre as importações de biblioteca e as importações de arquivos do projeto.
```suggestion

import { generatingQRCode } from '../../utils/generateQRCode.js';
```

Reply:
Certo